### PR TITLE
Update translator languages

### PR DESCRIPTION
**Pull request by saviit**
Updates the supported languages for automatic translations (DeepL).

New languages:
- Indonesian
- Korean
- Norwegian (bokmål)
- Turkish
- Ukrainian

After pulling locally, run:
`./tim run flask language update`
`./tim update all`

### CODE REVIEW

**Comment by vesal** [comment]
$ ./tim run flask language update
Usage: flask language [OPTIONS] COMMAND [ARGS]...
Try 'flask language --help' for help.

Error: No such command 'update'.
[ERROR] Command '['docker', 'compose', '-f', 'E:/tim/docker-compose.yml', 'run',
'--rm', '--workdir', '/service/timApp', 'tim', 'flask', 'language', 'update']'
returned non-zero exit status 2.

**Comment by saviit** [comment]
> Error: No such command 'update'. [ERROR] Command '['docker', 'compose', '-f', 'E:/tim/docker-compose.yml', 'run', '--rm', '--workdir', '/service/timApp', 'tim', 'flask', 'language', 'update']' returned non-zero exit status 2.

Mitä sulla sanoo `./tim run flask language` ? Tuon pitäisi listata kaikki komennot mitä tuolta language_cli:stä löytyy.

**Comment by vesal** [comment]
$ ./tim run flask language
Usage: flask language [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  add                Add a standard language to the language database...
  add-all-languages  Add languages defined in configuration into database...
  create             Creates a new custom language and adds it to the...
  remove             Remove a specific language from the database by...

**Comment by saviit** [comment]
> $ ./tim run flask language Usage: flask language [OPTIONS] COMMAND [ARGS]... Options: --help Show this message and exit. Commands: add Add a standard language to the language database... add-all-languages Add languages defined in configuration into database... create Creates a new custom language and adds it to the... remove Remove a specific language from the database by...
> […](#)
> On Wed, 22 Feb 2023, Sami Viitanen wrote: Error: No such command 'update'. [ERROR] Command '['docker', 'compose', '-f', 'E:/tim/docker-compose.yml', 'run', '--rm', '--workdir', '/service/timApp', 'tim', 'flask', 'language', 'update']' returned non-zero exit status 2. Mitä sulla sanoo ./tim run flask language ? Tuon pitäisi listata kaikki komennot mitä tuolta language_cli:stä löytyy. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.[AAGJYGERKPOEPAABOJ2NZYTWYXZOHA5CNFSM6AAAAAAVEGGTWOWGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTSV2JVIU.gif] Message ID: ***@***.***>

Ok, kokeilepa vielä `./tim update back` ennen tuota flask-komentoa.

**Comment by vesal** [comment]
$ ./tim update back
[INFO] Sending restart signal to Gunicorn
Gunicorn is not running, nothing to do!

***@***.*** MINGW64 /e/tim (master)
$ ./tim run flask language
Usage: flask language [OPTIONS] COMMAND [ARGS]...

Options:
  --help  Show this message and exit.

Commands:
  add                Add a standard language to the language database...
  add-all-languages  Add languages defined in configuration into database...
  create             Creates a new custom language and adds it to the...
  remove             Remove a specific language from the database by...

File: timApp/document/translation/language.py [hunk 1]
```diff
@@ -123,3 +123,13 @@ def to_json(self) -> dict:
             "flagUri": self.flag_uri,
             "name": self.autonym,
         }
+
+    # ignore type checking this for now, see: https://peps.python.org/pep-0484/#the-problem-of-forward-declarations
+    def __lt__(self, other) -> bool:  # type: ignore
```
Comment (dezhidki) [inline timApp/document/translation/language.py:128]:
```suggestion
    def __lt__(self, other: "Language") -> bool:
```

MyPy ja PyCharm tunnistavat tyypin, jos se on merkitty merkkijonona. Python 3.11:ssa sitten on erikseen Self-tyyppi, mutta sitä odotellessa voidaan käyttää tuota merkkijonoversiota.

**Comment by dezhidki** [comment]
> Commands:
>  add                Add a standard language to the language database...
>  add-all-languages  Add languages defined in configuration into database...
>  create             Creates a new custom language and adds it to the...
>  remove             Remove a specific language from the database by...

Oletko checkoutannut tämän PRn haaran? Itselläni ainakin toimii niin kuin Sami kuvasi.

**Review by dezhidki — APPROVED** [review]

**Pull request merged**